packaging/suse/patches/0_change_versions.gem.patch

```diff
@@ -4,19 +4,19 @@ index cd17fc91be2a..cd5d5580f36d 100644
 +++ b/Gemfile
 @@ -1,6 +1,6 @@
  source "https://rubygems.org"
- 
+
 -gem "rails", "~> 4.2.8"
 +gem "rails", "~> 4.2.2"
```

Comment:
Yes, this patch is a bit of a mess 😅 This is for the final RPM, in which we have these old versions but we have included all the security patches in them, so security-wise it should be the same as 4.2.8. That being said, for this specific gem we are working on the upgrade, so we don't have to move around patches 👍